include/ttmlir/Dialect/TTNN/IR/TTNNOps.td

```diff
@@ -255,6 +255,19 @@ def TTNN_LogicalNotOp: TTNN_ElementwiseUnaryOp<"logical_not"> {
     }];
 }
 
+def TTNN_BitwiseNotOp : TTNN_ElementwiseUnaryOp<"bitwise_not"> {
```

Comment:
```c
    static Tensor invoke(
        const Tensor& input_tensor,
        const std::optional<MemoryConfig>& memory_config = std::nullopt,
        const std::optional<Tensor>& optional_output_tensor = std::nullopt);
```

This op has an optional memory config operand as well, we should add it here. All the other ops that were added should have their operands match signatures from ttnn lib.

Reply:
In an offline agreement, we decided to raise separate [issue](https://github.com/tenstorrent/tt-mlir/issues/1612) for this, and leave things as is in this PR.